### PR TITLE
feature/signup: adds signup screen

### PR DESCRIPTION
**Pull request by hamxabaig**

### CODE REVIEW

File: src/components/screens/LoginScreen/index.js [hunk 1]
```diff
@@ -30,31 +27,39 @@ class LoginScreen extends Component {
   };
 
   render() {
-    const { loading, error, errorMessage } = this.props;
+    const { loading, error, errorMessage, registerUserSuccess } = this.props;
```
Comment (mraafaykhan) [inline src/components/screens/LoginScreen/index.js]: please elaborate what the app expects from the api as a result. 
Are you going to assign these props in the controller, 
like normally a api would return something like
on successful registration res :{
 success: true, 
jwt: ....... ( or we could as the user to login again). 
}

File: src/reducers/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import auth from './auth/auth.reducer';
 import app from './app/app.reducer';
 
-export const API_URL = 'http://ec2-13-250-2-39.ap-southeast-1.compute.amazonaws.com/api';
+export const API_URL = 'http://phototype.me/api';
 export const BUCKET_URL = 'https://s3.ap-south-1.amazonaws.com/phototype';
```
Comment (mraafaykhan) [inline src/reducers/index.js]: why do we need the bucket url on the client app ?

**Comment by mraafaykhan** [comment]
kindly discuss the api structure the app is expecting, we should make an api sheet, so that the web and app end are expecting/sending the same thing.

File: src/components/screens/LoginScreen/index.js (hunk 1 above)
Comment (hamxabaig) [reply src/components/screens/LoginScreen/index.js]: @mraafaykhan its in reducer, Its not coming from API. Reducer checks if api returns `success`

File: src/reducers/index.js (hunk 2 above)
Comment (hamxabaig) [reply src/reducers/index.js]: Coz we're not gonna store the bucket path in db. So this acts as base url for the path.

**Comment by hamxabaig** [comment]
@mraafaykhan We're using swagger, It generates API documentation automagically from the routes. The link phototype.me/documentation is not working atm. I'll check it.
If you need to document anything, you can use `notes` or `description` keys in the route.js file.

**Pull request merged**